multiqc/modules/odgi/odgi.py

```diff
@@ -27,7 +27,7 @@ def __init__(self):
             name="ODGI",
             anchor="odgi",
             href="https://github.com/pangenome/odgi",
-            info="is an optimized dynamic graph/genome implementation.",
+            info="is an optimized dynamic graph/genome implementation, allowing efficient computation and transformation of pangenomes with minimal overheads.",
```

Comment:
I would not formulate it like this. Rather:
`, for efficient analysis and manipulation of pangenome graphs structured in the variation graph model`.
I will push something here.

Reply:
Done.